snap: add example to update to another channel (#6046)
tldr-pages/tldr: pages/linux/snap.md

`snap refresh {{package_name}}`

> - Update a package to another channel (track, risk-level, or branch):

Reviewer: What is `track, risk-level, or branch`?
```suggestion
- Update a package to another channel:
```

Author: I wasn't sure if the term channel was going to widely understood enough, so I thought that might help make it more explicit.

Track: A type of release, by default it's `latest`, I've never seen this used in any way tbh.
Risk-Level: Stable / Beta / Edge
Branch: For debug releases, like a one-off version of a particular track/risk-level to fix a particular issue or that allows a user to test a specific new feature without giving it to everyone.

I'd like to wait for other opinions, if they agree to remove it, then that's cool.

Reviewer: Maybe
```suggestion
- Update a package to another channel (e.g. stable, beta or development):
```
Not sure if we should explain the concept of `channel`.

Reviewer 2: What are the possible valid values?

Reviewer: It's not fixed, the publisher can use whatever their wants. Node uses `14/stable`, intellij `edge`, `beta`, `stable`.

Reviewer 2: Ah, I was afraid so.  No suggestions on my part, then :(

Reviewer 3: Generally here we want to use the official terminology that the application itself uses

Author: I assume that means we're deciding between leaving it at channel, or specifying track, risk-level, or branch?

Opinion on each?
1️⃣ Update a package to another channel:
2️⃣ Update a package to another channel (track, risk-level, or branch):
3️⃣ Seth, you misunderstood. Your ideas are bad, and you should feel bad.

Author: Ahh so sad, I was hoping people could react with 1️⃣, 2️⃣, and 3️⃣. Seems we're limited to just the 8 GitHub give us.

Reviewer 2: Yeah, GitLab is much better in the regard 😢

Reviewer 3: What's the official terminology used by the snapcraft website?

Author: ![image](https://user-images.githubusercontent.com/22801583/119654604-eed77900-be28-11eb-8d4e-42f368278a29.png)

In the docs I linked in the PR description

Reviewer: e.g. `14/stable` for Node.js and `latest/beta`, `2021.1/stable` for IntelliJ. Cool. To make it explicit: The combination of `<track>/<risk>/<branch>` is the name of the channel.

Reviewer 3: In that case, we should probably use that terminology then, to avoid confusing things.

Author: So, we're happy with something like the following?

```suggestion
- Update a package to another channel (track, risk, or branch):
```

Reviewer 3: Leaving unresolved on purpose, as this looks like an important conversation.
